### DISCUSSION TITLE
What is the `manifest.json` created in build `/app/manifest.json`?

### DISCUSSION THREAD

**Discussion by frederikhors** [sveltejs/kit#3271] [Q&A]
I'm using adapter-static.

What is the `manifest.json` created in build `/app/manifest.json`?

Do I need it?

It starts like this:

```json
{
  ".svelte-kit/build/runtime/internal/start.js": {
    "file": "start-4a47708e.js",
    "src": ".svelte-kit/build/runtime/internal/start.js",
    "isEntry": true,
    "imports": [
      "_vendor-660c66ca.js",
      "_singletons-a42a5e91.js"
    ],
    "dynamicImports": [
      "src/routes/__layout.svelte",
      "src/routes/__error.svelte",
      "src/routes/index.svelte",
...
```

**Comment by yeldiRium** [comment]
Bumping this question, since the `manifest.json` blocks creating a `manifest.json` for PWAs. (I could put my own `manifest.json` somewhere else, but if the generated one is avoidable, I'd rather put the file in the generated site's root.)

**Reply by yeldiRium** [reply]
After reading [this](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Add_to_home_screen) I retract my comment, since the extension `.webmanifest` is recommended for a PWA manifest. So my point is irrelevant to this discussion.

**Comment by Rich-Harris** [comment]
It's something Vite creates, that we use (among other things) to figure out which client-side files are needed for which route. It's not supposed to be publicly accessible, this is an unintended result of a recent change. See https://github.com/sveltejs/kit/pull/5874
